src/passes/constants.ts

```diff
@@ -53,7 +53,7 @@ declare class _PeggySyntaxError extends Error {
   public name: string;
   constructor(message: string, expected: Expectation[], found: string | null, location: FileRange);
   format(sources: {
-    grammarSource?: string;
+    source?: any;
```

Comment:
Yes, this looks right.  It has to be `any` because you can pass a complicated object in there that knows about start offsets, etc.